### PR TITLE
Add different html files for the pages

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,42 @@
...
+          <li>tag 1</li>
+          <li>tag 2</li>
+          <li>tag 3</li>
+          <li>tag 4</li>
+        </ul>
+      </article>
+      <nav>
+        <ul>
+          <li>
+            <a href="./index.html" aria-label="change to quiz-app">Quiz-App</a>
```
Comment (ttaannjjaa) [inline index.html:30]: very good for my opinion to add the aria-labels

File: index.html [hunk 2]
```diff
@@ -4,10 +4,42 @@
...
+    <title>Quiz App Start</title>
     <link rel="stylesheet" href="./styles/index.css" />
   </head>
   <body>
     <h1>Quiz App</h1>
+    <main>
+      <article>
+        <h2>Question</h2>
+        <p>Text of question</p>
+        <button>show answer</button>
```
Comment (ttaannjjaa) [inline index.html]: just to let you know: i decided to add here a second button for hiding the answer, but when i am thinking now about it, probably this problem can be solved by adding 2 classes with different modifiers for active and not-active

File: index.html [hunk 3]
```diff
@@ -4,10 +4,42 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz App Start</title>
     <link rel="stylesheet" href="./styles/index.css" />
   </head>
   <body>
     <h1>Quiz App</h1>
+    <main>
+      <article>
```
Comment (ttaannjjaa) [inline index.html:13]: suggestion to add also some html for that bookmark icon or image around here

File: profile.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Profile</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Profile</h1>
+    <main>
+      <section>
+        <img src="" alt="" width="200" height="150" />
```
Comment (ttaannjjaa) [inline profile.html]: for semantic html i think it is better to add a `figure` tag to wrap the `img `
if this shall be indeed a circle maybe it is better to set width and height at the same value

File: create.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Create</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Create</h1>
+    <main>
+      <form aria-labelledby="create-question-heading" autocoplete="off">
```
Comment (ttaannjjaa) [inline create.html]: 😊 little typo..autocomplete

as i understood the wireframe it is not necessary to give a new question a heading , so you could shorten the aria-labelledby value to "create-question"

File: bookmarks.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Bookmarks</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Bookmarks</h1>
+    <main>
+      <article>
+        <h2>Question</h2>
```
Comment (ttaannjjaa) [inline bookmarks.html:14]: i think here is also the "bookmark-img" or icon necessary ☺️

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Great and fast work 👍, much more than requested 😃

File: bookmarks.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Bookmarks</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Bookmarks</h1>
```
Comment (MichiBerger) [inline bookmarks.html:11]: cool jelena...easy and simple 👍 i would recommend to wrap your `h1` in a `header`  tag

File: create.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+        </textarea>
+        <label for="input-tags">Please name tags:</label>
+        <textarea
+          id="input-tags"
+          cols="40"
+          rows="1"
+          required
+          placeholder="Tag 1, Tag 2, Tag 3, Tag 4"
+        >
+        </textarea>
```
Comment (MichiBerger) [inline create.html]: here i would recommend more an `input` instead of an `textarea`

File: index.html (hunk 2 above)
Comment (MichiBerger) [reply index.html]: good point tanja...that actually makes sense maybe with a liitle bit of javascript at the end

File: index.html (hunk 1 above)
Comment (JelenadlS) [reply index.html:30]: Thank you, I thought it makes sense this time 😊

File: index.html (hunk 2 above)
Comment (JelenadlS) [reply index.html]: I also first thought about this and then googeled and I think it will be created with java 😊

File: index.html (hunk 3 above)
Comment (JelenadlS) [reply index.html:13]: haha also this one I first thought so but also understood to do it with java - but im not sure about this as i do not know anything about java 😅

File: profile.html (hunk 4 above)
Comment (JelenadlS) [reply profile.html]: oh thank you for the advise, you are totally right! 😊

File: create.html (hunk 5 above)
Comment (JelenadlS) [reply create.html]: wow thank you Tanja, you are really looking into details - great 👍🏼

File: bookmarks.html (hunk 7 above)
Comment (JelenadlS) [reply bookmarks.html:11]: Thank you 😊
I first had it, but Jerry commented in Andreas pull request that it is not necessary when just having one thing in it - 
Thats why I deleted it.🤷🏼‍♀️

File: create.html (hunk 8 above)
Comment (JelenadlS) [reply create.html]: Oh thank you Michi, I think you are absolutely right 😊

**Pull request merged**